python-checks/src/main/java/org/sonar/python/checks/FilePermissionsCheck.java

```diff
@@ -75,21 +76,29 @@ private static void checkSensitiveArgument(List<Argument> arguments , int sensit
       return;
     }
     Expression expression = modeArgument.expression();
+    if(isUnsafeExpression(expression, safeModulo)) {
+      ctx.addIssue(modeArgument, MESSAGE);
+    }
+  }
+
+  private static boolean isUnsafeExpression(Expression expression, int safeModulo) {
     if (expression instanceof HasSymbol) {
       Symbol symbol = ((HasSymbol) expression).symbol();
       if (symbol != null && SENSITIVE_CONSTANTS.contains(symbol.fullyQualifiedName())) {
-        ctx.addIssue(modeArgument, MESSAGE);
-        return;
+        return true;
       }
     }
+    if (expression.is(Tree.Kind.BITWISE_OR)) {
```

Comment:
Can I nitpick? :)
We have a FN for the following case:
```
x = stat.S_IRWXU | stat.S_IRWXG | stat.S_IRWXO
os.chmod("/tmp/fs", x)
```
In fact, we also have a FN for a simpler case:
```
x = stat.S_IRWXO
os.chmod("/tmp/fs", x)
```